execution/executor-service/src/remote_state_view.rs

```diff
@@ -114,25 +123,61 @@ impl RemoteStateViewClient {
 
     pub fn init_for_block(&self, state_keys: Vec<StateKey>) {
         *self.state_view.write().unwrap() = RemoteStateView::new();
-        self.pre_fetch_state_values(state_keys);
+        info!(
```

Comment:
nit: remove this log line as this is going to be spammy

Reply:
ok